integreat_cms/release_notes/current/unreleased/2621.yml

```
@@ -0,0 +1,2 @@
+en: Outdate HIX value when reversing changes in the editor
+de: Veralte HIX-Wert beim Rückgängigmachen von Änderungen im Editor
```

Not sure if it sounds okay in German?

---

Maybe like this?
```suggestion
de: Markiere HIX-Wert als veraltet, wenn Änderungen im Editor rückgängig gemacht werden
```

Also the English text could be made a bit more clear, but I would also be fine with the current wording.
```
en: Mark HIX value as outdated when changes are undone in the editor
```